src/scss/parse.js

```diff
@@ -2767,7 +2769,12 @@ module.exports = (function() {
         var startPos = pos,
             x = [];
 
-        x.push(checkVariable(pos) ? getVariable() : getIdent());
+        while (pos < tokensLength) {
```

Comment:
I think there (and above) should be no loop: I can't think of an example when interpolation is followed by variable.